specs/payment-api.yaml

```diff
@@ -1445,3 +1486,30 @@ components:
       required:
         - platform
         - address
+
+    MarketDecoration:
```

Comment:
There is `DemandOfferBase` defined in Market API. Consider reusing it. We can move it to `common.yaml` if needed.

Reply:
@tworec I agree that re-using a common base class would be a good idea, but this PR is really urgent and I would like to merge it ASAP. Also, I am not quite convinced which design is better – serialized dictionary (JSON–in–JSON) vs. explicit key-value list. Therefore, please forgive me if I merge this without applying your suggestion.

Reply:
no worries